doc/source/gotcha.rst

```diff
@@ -1,20 +1,94 @@
 .. currentmodule:: pandas
-.. _gotchas:
+.. _faq:
```

Comment:
keep this as `gotchas`? as the file is still like that?

Also, I would keep the filename intact (just to prevent possible changes in links that are not necessary)